Add Cancel Query section to DBM docs (#29688)
DataDog/documentation: content/en/database_monitoring/setup_postgres/advanced_configuration.md

    query_samples:
        collection_interval: 0.1
```

> ## Setup Cancel Query integration

Reviewer: ```suggestion
## Set up Cancel Query integration
```

> In order to support cancelling a query throughout the Database Monitoring product, you need to setup the Private Actions Runner. Follow Datadog documentation on how to setup a Private Actions runner in your environment: https://docs.datadoghq.com/actions/private_actions/use_private_actions/?tab=docker#set-up-a-private-action-runner. You must enable the “Cancel Query” action for this runner.

Reviewer: ```suggestion
To support cancelling a query throughout the Database Monitoring product, [set up a private action runner][1] and enable the “Cancel Query” action for the runner.
```

The runner runs using an existing user in the database, the user is required to have `pg_signal_backend` and `pg_read_all_stats` permissions in the database.

> ### Create a Connection to the Private Actions Runner

Reviewer: ```suggestion
### Create a Connection to the private action runner
```

Reviewer: We can keep it capitalized. Generally, we use the singular in the docs, though: "Private Action Runner". So I'd like to keep it consistent

Create a Postgres connection in Datadog Actions Connections, and name and configure the connection:

> {{< img src="database_monitoring/db-cancel-query-connection-modal.png" alt="The Connection creation modal for the Cancel Query feature" style="width:90%;" >}}

Reviewer: ```suggestion
{{< img src="database_monitoring/db-cancel-query-connection-modal.png" alt="The Connection creation modal for the Cancel Query feature" style="width:90%;" >}}

[1]: /actions/private_actions/use_private_actions/#set-up-a-private-action-runner
```
